### DISCUSSION TITLE
Bot not playing Audio

### DISCUSSION THREAD

**Discussion by Tuscani712** [jagrosh/MusicBot#827] [Troubleshooting] [answered]
I'm having some issues getting a the bot to play while being hosted on a cloud server.

Using Linode as a host.

I can host and play fine from my home server, however now that i've tried to migrate it to an online based service, the bot will join the channel and light up like it is playing but no sound will come out.

Using 0.3.5
Hosting on Ubuntu 20.04 virtual machine.

Music bot console output
```
[19:19:42] [ERROR] [Startup]: Could not start GUI. If you are running on a serve|
r or in a location where you cannot display a window, please run in nogui mode u|
sing the -Dnogui=true flag.                                                     |
[19:19:42] [INFO] [Startup]: Loaded config from /home/musicbot/config.txt       |
[19:19:43] [INFO] [JDA]: Login Successful!                                      |
[19:19:43] [INFO] [WebSocketClient]: Connected to WebSocket                     |
[19:19:43] [INFO] [JDA]: Finished Loading!                                      |
[19:20:09] [INFO] [NativeLibraryLoader]: Native library libmpg123-0: loading wit|
h filter com.sedmelluq.discord.lavaplayer.natives.ConnectorNativeLibLoader$$Lamb|
da$260/0x0000000840315040@21c99428                                              |
[19:20:09] [INFO] [NativeLibraryLoader]: Native library connector: loading with |
filter null                                                                     |
[19:20:09] [INFO] [NativeLibraryLoader]: Native library connector: successfully |
loaded.
```

Debug output
```
System Properties:
  java.version = 11.0.11
  java.vm.name = OpenJDK 64-Bit Server VM
  java.vm.specification.version = 11
  java.runtime.name = OpenJDK Runtime Environment
  java.runtime.version = 11.0.11+9-Ubuntu-0ubuntu2
  java.specification.version = 11
  os.arch = amd64
  os.name = Linux

JMusicBot Information:
  Version = 0.3.5
  Owner = XXXXXXXXXXXXXXXXXX
  Prefix = @mention
  AltPrefix = null
  MaxSeconds = 0
  NPImages = true
  SongInStatus = true
  StayInChannel = true
  UseEval = false
  UpdateAlerts = true

Dependency Information:
  JDA Version = 4.3.0_312
  JDA-Utilities Version = 3.0.5
  Lavaplayer Version = 1.3.78

Runtime Information:
  Total Memory = 17
  Used Memory = 10

Discord Information:XXX
  ID = XXXXXXXXXXXXXXXX
  Guilds = 1
  Users = 2
```

I've tried different version of java but none seem to be helping.

Let me know what other information I can provide to help troubleshoot my issue.

Thanks!

**Comment by MichailiK** [comment]
Play something, then use the `nowplaying` command. If you see the playback progressing (the messags updates itself every 5 seconds), then there's likely something wrong with your Discord audio settings.

**Reply by Tuscani712** [reply]
The time shows progression when using the command. Also, the bot previously worked with the same exact discord settings. (Bot role and permissions etc, as well as the server has not changed region)

Thanks for trying to help.

**Comment by Tuscani712** [comment, marked as answer]
Issue resolved.

Sometime after I migrated the server my discord took a dump. Joined in today and couldn't hear any of my friends. After some troubleshooting, ended up re-installing my head set driver. I could hear discord noises and youtube videos, but could not hear any voices in discord. Tested again with bot and seems to be working.

Apologies for the stupid thread.

tldr; bot was working, discord needed me to reinstall headset driver.
